database/index.js

```diff
@@ -59,5 +61,36 @@ var getReviewsByHotel = (hotelId) => {
   })
 }
 
+var update = (updateInfo) => {
+  return new Promise((resolve, reject) => {
+    Review.findOneAndUpdate({_id: updateInfo.id},
+      {$set:{'responderInfo.responderText': updateInfo.newText}},
+      {new: true},
+      (err, results) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(results);
+      }
+    })
+  })
+}
+
+var deleteRev = (reviewId) => {
+  return new Promise((resolve, reject) => {
+    Review.findOneAndRemove({_id: reviewId}, (err, results) => {
+      if (err) {
+        console.log("could not delete");
+        reject(err);
+      } else {
+        console.log("deleted");
+        resolve(results);
+      }
+    })
+  })
+}
+
 module.exports.save = save;
-module.exports.getReviewsByHotel = getReviewsByHotel;
\ No newline at end of file
+module.exports.getHotelReviews = getHotelReviews;
+module.exports.update = update;
+module.exports.deleteRev = deleteRev;
```

Comment:
### Destructuring Rules!
You could save some space by substituting this code...
```
module.exports = { save, getHotelReviews, update, deleteRev };
```